lib/services/elasticsearch.js

```diff
@@ -812,7 +816,7 @@ class ElasticSearch extends Service {
     const exists = await this.collectionExists(index, collection);
 
     if (exists) {
-      return this.updateMapping(index, collection, mappings);
+      return this.updateCollection(index, collection, { mappings, settings });
```

Comment:
Shouldn't we throw an "already exists" error instead?

Reply:
I'm afraid it will be a breaking change since we allow subsequent calls to `collection:create` to modify the mappings :-/
 (I fact we use this behavior in the core also in the [bootstrap sequence](https://github.com/kuzzleio/kuzzle/blob/master/lib/core/storage/bootstrap/internalIndexBootstrap.js#L96) )